client/app/utils/ember-changeset.js

```diff
@@ -0,0 +1,9 @@
+export function addToHasMany(changeset, relationship, item) {
+  let hasMany = changeset.get(relationship).pushObjects([item]);
+  changeset.set(relationship, hasMany.toArray());
+}
+
+export function removeFromHasMany(changeset, relationship, item) {
+  let hasMany = changeset.get(relationship).removeObject(item);
+  changeset.set(relationship, hasMany.toArray());
+}
```

Comment:
This is a cool approach!

Reply:
Cribbed from here: https://davidtang.io/2018/07/28/tracking-has-many-relationship-changes-with-ember-changeset.html

Reply:
Oh yeah I remember reading about this a while back, looks great!